### PR TITLE
Modified user-guide on kompose.io

### LINKED ISSUE #479: Update unsupported keys in kompose.io user guide
Remove keys which are supported now: `build`, `mem_limit`, `tty`, `stdin_open`, `user`

Add unsupported keys: `cap_add`, `cap_drop`, `sysctls`

### PR DESCRIPTION
**Pull request by surajnarwade**
Fixes #479

### CODE REVIEW

**Comment by kadel** [comment]
I have stupid question :blush: Do we even need this list here? Is this duplicating what is already in [conversion.md](https://github.com/kubernetes-incubator/kompose/blob/master/docs/conversion.md)

I think we can just link to that document. It will be easier to keep it in one place

**Comment by cdrage** [comment]
I agree with @kadel :)

**Comment by surajnarwade** [comment]
@cdrage  @kadel , I thought of updating list on [kompose.io](http://kompose.io/)

**Comment by cdrage** [comment]
@surajnarwade @kadel 

Yes, what @kadel is saying is removing it from both docs/user-guide.md in master as well as in gh-pages.

**Comment by kadel** [comment]
> Yes, what @kadel is saying is removing it from both docs/user-guide.md in master as well as in gh-pages.

yes, and replacing it with link to conversion table

**Comment by surajnarwade** [comment]
got it :)

File: docs/user-guide.md [hunk 1]
```diff
@@ -298,11 +298,7 @@ The chart structure is aimed at providing a skeleton for building your Helm char
 
 ## Unsupported docker-compose configuration options
 
-Currently `kompose` does not support the following Docker Compose options.
-
-```
-"build", "cgroup_parent", "devices", "depends_on", "dns", "dns_search", "domainname", "env_file", "extends", "external_links", "extra_hosts", "hostname", "ipc", "logging", "mac_address", "mem_limit", "memswap_limit", "network_mode", "networks", "pid", "security_opt", "shm_size", "stop_signal", "volume_driver", "uts", "read_only", "stdin_open", "tty", "user", "ulimits", "dockerfile", "net"
-```
+Currently `kompose` does not support some of the Docker Compose options, which are listed on [conversion](http://kompose.io/conversion/) page.
```
Comment (cdrage) [inline docs/user-guide.md]: `some Docker Compose options` 

`listed on the conversion document`

**Comment by cdrage** [comment]
Oh, and update the commit message as well please!

**Comment by surajnarwade** [comment]
@cdrage , updated both

**Comment by cdrage** [comment]
@surajnarwade Thanks, the git commit message still isn't modified however.

**Comment by surajnarwade** [comment]
@cdrage , my bad, updated now :)

**Comment by cdrage** [comment]
LGTM! :+1:

**Pull request merged**